packages/frontend/src/components/common/modal/DashboardCreateModal.tsx

```diff
@@ -118,111 +110,81 @@ const DashboardCreateModal: FC<DashboardCreateModalProps> = ({
-                            label="Name your space"
-                            labelFor="dashboard-space"
-                        >
-                            <InputGroup
-                                id="dashboard-space"
-                                type="text"
-                                value={newSpaceName}
-                                onChange={(e) =>
-                                    setNewSpaceName(e.target.value)
-                                }
+                            <Button
+                                minimal
+                                icon="plus"
+                                intent="primary"
+                                onClick={() => setIsCreatingNewSpace(true)}
+                            >
+                                Create new space
+                            </Button>
+                        </>
+                    ) : (
+                        <>
+                            <Input
+                                label="Name your space"
+                                name="newSpaceName"
                                 placeholder="eg. KPIs"
+                                rules={{ required: 'Name field is required' }}
                             />
-                        </FormGroup>
-                        <Button
-                            minimal
-                            icon="arrow-left"
-                            intent="primary"
-                            onClick={() => setIsCreatingnewSpace(false)}
-                        >
-                            Save to existing space
-                        </Button>
-                    </>
-                )}
-            </DialogBody>
-
-            <DialogFooter
-                actions={
-                    <>
-                        <Button
-                            onClick={(e) => {
-                                handleClose(e);
-                                if (onClose) onClose(e);
-                            }}
-                        >
-                            Cancel
-                        </Button>
-                        <Button
-                            intent={Intent.PRIMARY}
-                            text="Create"
-                            type="submit"
-                            loading={isCreatingDashboard || isCreatingSpace}
-                            onClick={handleConfirm}
-                            disabled={
-                                (isCreatingDashboard && dashboardName === '') ||
-                                (isCreatingNewSpace && newSpaceName === '')
-                            }
-                        />
-                    </>
-                }
-            />
+                            <Button
+                                minimal
+                                icon="arrow-left"
+                                intent="primary"
+                                onClick={() => setIsCreatingNewSpace(false)}
```

Comment:
I think this needs to be type `submit`...

Reply:
it doesn't submit the form though 🤔 it's just to go back to the select instead of input

Reply:
ignore! 🤦